packages/atomic/src/components/common/interface/interface-common-stencil.tsx

```diff
@@ -39,7 +39,7 @@ export class CommonAtomicInterfaceHelper<Engine extends AnyEngineType> {
     private atomicInterface: StencilBaseAtomicInterface<Engine>,
     globalVariableName: string
   ) {
-    setCoveoGlobal(globalVariableName);
+    setCoveoGlobal(VERSION, globalVariableName);
```

Comment:
I'm guessing we still inject the version in a common here because it's going away post-migration, correct?
Otherwise, I'm unsure about the fix.